### PR TITLE
Fix some language nits

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: draft-iab-rfcefdp-rfced-model.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ positions as described below.
 
 Because policy issues can be difficult and contentious, RSWG
 participants and RSAB members are strongly encouraged to work together 
-in a spirit of good faith and mutual understanding to achieve rough 
+in a spirit of good faith and mutual understanding to achieving rough
```
Comment (elear) [inline draft-iab-rfcefdp-rfced-model.md:296]: I don't think this one's correct.

File: draft-iab-rfcefdp-rfced-model.md [hunk 2]
```diff
@@ -386,11 +386,11 @@ Notices will always be sent to the rfc-interest mailing list. The RSAB and
 RSWG should also send notices to other communities that may be interested 
 in or impacted by a proposal as they see fit, following policies for those 
 fora as appropriate. Notices are also to be made available and archived on 
-the rfc-editor.org web site, and other communication channels can be 
+the rfc-editor.org website, and other communication channels can be
```
Comment (elear) [inline draft-iab-rfcefdp-rfced-model.md:389]: Astonishingly, Lars' edit here is in accordance with Miriam Webster!!  Who knew??

**Review by elear — CHANGES_REQUESTED** [review]

File: draft-iab-rfcefdp-rfced-model.md (hunk 2 above)
Comment (stpeter) [reply draft-iab-rfcefdp-rfced-model.md:389]: RFC 7322 (the RFC Style Guide) uses "website" so I suggest we keep this as-is.

**Comment by stpeter** [comment]
I've made the appropriate changes in my local copy, so I'm going to close this one without merging.

**Pull request closed without merging**